index.css

```diff
@@ -0,0 +1,542 @@
+}
+
+.contact-layout__title {
+  left: 0px;
+  top: 64px;
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 57px;
+  letter-spacing: 0.2px;
+  color: #252b42;
+}
+
+.contact-layout__input-container {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  margin-top: 2em;
+}
+.contact-layout__input-container1 {
+  display: flex;
+  gap: 10px;
+}
+.contact-layout__input {
+  box-sizing: border-box;
+  background: #f9f9f9;
+  border: 1px solid #bdbdbd;
+  border-radius: 5px;
+  width: 285px;
+  height: 50px;
+}
+.contact-layout__input-messages {
+  width: 570px;
+  height: 140px;
+  background: #f9f9f9;
+  border: 1px solid #bdbdbd;
+  border-radius: 5px;
+  padding-left: 10px;
+}
+.button--button {
+  padding: 15px 30px;
+  gap: 10px;
+  width: 218px;
+  height: 58px;
+  border-radius: 5px;
+  font-size: 14px;
+}
+.contact-layout__Agency {
+  margin-top: 10rem;
+  display: flex;
+  align-items: center;
+  background: #fafafa;
+  padding: 20px 0 0 5rem;
+}
+.contact-layout__Agency-button {
+  cursor: pointer;
+  outline: none;
+  border: none;
+  border-radius: 5px;
+  background-color: #ffa62b;
+  width: 120px;
+  height: 48px;
+  color: white;
+  margin-left: 38rem;
+}
+.contact-layaut__Agency-content {
+  font-family: "Montserrat";
+  padding: 1rem 0 0 6rem;
+}
+
+.contact-layout__Agency-title {
+  font-size: 24px;
+  color: #252b42;
+}
+.contact-layout__Agency-subtitle {
+  font-size: 14px;
+  color: #737373;
+}
+.footer {
+  background-color: #252b42;
+  color: white;
+}
+
+.footer__content-container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  padding: 2rem 9rem ;
+}
+
+.footer__contents {
+  display: flex;
+  justify-content: space-between;
+}
+.footer__logo {
+  font-weight: 700;
+  font-size: 24px;
+  color: white;
+}
+
+.footer__socials {
+  display: flex;
+  gap: 15px;
+}
+
+.footer__border {
+  width: 100%;
+  border: 1px solid #364067;
+  margin: 50px 0;
+}
+
+.footer__menu-item {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+}
+
+.footer__menus {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.footer__title {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0.1px;
+  margin-bottom: 10px;
+}
+
+.footer__menu {
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 24px;
+  letter-spacing: 0.2px;
+  color: #bdbdbd;
+  cursor: pointer;
+}
+
+.footer__Emails{
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid white;
+  display: flex;
+}
+
+.footer__input {
+  display: block;
+  background: none;
+  border: none;
+  outline: none;
+  padding: 0 6px;
+  height: 100%;
+}
+.footer__menu--sm {
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.footer__author-container{
+  background: #16697a;
+  color: white;
+  padding-bottom: 25px;
+  font-size: 14px;
+  line-height: 24px;
+  padding: 2rem 9rem ;
+  font-family: "Montserrat";
+}
```

Comment:
The name of your CSS is confusing. Give it another name "style.css" or other name which differs from it's index.html